homeworks/Tatiana.Skachko_S3tt0n/homework_6/objects.js

```diff
@@ -0,0 +1,31 @@
+const user = {};
+
+Object.defineProperty(user, 'fullName', {
+    configurable: true,
+    enumerable: true,
+    get() {
+        return this.capitName;
+    },
+    set(value) {
+        this.capitName = value.toLowerCase();
+        this.capitName = this.capitName.split(' ').map(name => name[0].toUpperCase() + name.substring(1)).join(' ');
+        this.capitName = this.capitName.split('-').map(name => name[0].toUpperCase() + name.substring(1)).join('-');
```

Comment:
two identical lines of spaghetti code... may you could prepare a function smth like `firstCharCapitalizer`?

also, this is a suiting case for RegExp